test/e2e/suites/managed/addon.go

```diff
@@ -62,8 +63,9 @@ func CheckAddonExistsSpec(ctx context.Context, inputGetter func() CheckAddonExis
 
 	By(fmt.Sprintf("Getting control plane: %s", controlPlaneName))
 	controlPlane := &ekscontrolplanev1.AWSManagedControlPlane{}
-	err := mgmtClient.Get(ctx, crclient.ObjectKey{Namespace: input.Namespace.Name, Name: controlPlaneName}, controlPlane)
-	Expect(err).ToNot(HaveOccurred())
+	Eventually(func() error {
+		return mgmtClient.Get(ctx, crclient.ObjectKey{Namespace: input.Namespace.Name, Name: controlPlaneName}, controlPlane)
+	}, 2*time.Minute, 5*time.Second).Should(Succeed(), "eventually failed trying to get the AWSManagedControlPlane")
```

Comment:
Could we perhaps use https://github.com/kubernetes-sigs/cluster-api/blob/main/test/framework/clusterctl/e2e_config.go#L769 and https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/main/test/e2e/data/e2e_conf.yaml so that the settings are in the same place as other `Eventually` timeout definitions?

https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/main/test/e2e/suites/managed/control_plane.go#L77 is an example of it being used with one of CAPA's helpers, and https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/main/test/e2e/suites/unmanaged/unmanaged_functional_test.go#L956 is one where it's used directly with `Eventually`

Reply:
Makes sense, Done.